src/features/Download/Cta/index.tsx

```diff
@@ -21,7 +21,7 @@ const firefoxDownloadHref = "/tally_ho_pre_release_channel-0.13.1-fx.xpi";
 export function DownloadCta() {
   return (
     <div
-      className={css`
+      className={"matomo_download " + css`
```

Comment:
I'm totally not aware of how matomo is working but are you sure adding this class to div will work the same way as with links elements?

Reply:
https://developer.matomo.org/guides/tracking-javascript-guide#recording-a-click-as-a-download

Yeah you can do it that way too but this is the functionality that LJ wants to implement. Looks like the Matomo script picks up this class by default and ends up with the same result essentially

Reply:
Why not add it in the Option component's `a` tag though?